src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/ReleaseTrackingAnalyzerTests.cs

```diff
@@ -55,6 +55,65 @@ class MyAnalyzer : DiagnosticAnalyzer
             await VerifyCSharpAsync(source, shippedText, unshippedText);
         }
 
+        [Fact]
+        public async Task TestCodeFixToEnableAnalyzerReleaseTracking()
+        {
+            var source = @"
+using System;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+class MyAnalyzer : DiagnosticAnalyzer
+{
+    private static readonly DiagnosticDescriptor descriptor1 =
+        new DiagnosticDescriptor({|RS2008:""Id1""|}, ""Title1"", ""Message1"", ""Category1"", DiagnosticSeverity.Warning, isEnabledByDefault: true);
+
+    public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(descriptor1);
+    public override void Initialize(AnalysisContext context) { }
+}";
+
+            var fixedSource = @"
```

Comment:
❔ Is this the same as `source`? If so, it would be good to just have one copy.

Reply:
Input source has a diagnostic.

Reply:
If it's a diagnostic the code fix claims to be able to fix, the checks will ignore that markup diagnostic during validation of the fixed state. This is because the default `MarkupHandling` value is `IgnoreFixable`:

https://github.com/dotnet/roslyn-sdk/blob/0408271325bb067fd5f84592a545e8280cedff39/src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/SolutionState.cs#L65-L70

Reply:
Ah thanks - I have made this change.